### PR TITLE
chore(npm): cleaner error logs

### PR DESCRIPTION
**Pull request by Haroenv**
right now we say "something went wrong asking for the downloads of", but there's actually downloads, scoped downloads and dependents,  which are three separate things

### CODE REVIEW

File: src/npm.js [hunk 1]
```diff
@@ -56,14 +56,28 @@ export async function getDownloads(pkgs) {
     ...pkgsNamesChunks.map(pkgsNames =>
       got(`${c.npmDownloadsEndpoint}/point/last-month/${pkgsNames}`, {
         json: true,
-      }).catch(e => suppressError(e, pkgsNames, { body: {} }))
+      }).catch(error =>
+        suppressError({
```
Comment (vvo) [inline src/npm.js]: Can we use: suppressError => logWarning and things => type?

Comment (Haroenv) [reply src/npm.js]: Good idea, will refactor :)

**Comment by vvo** [comment]
Would call it logWarning since we do not consider this an error, just a warning. I'd consider anything an error something that we cannot recover from (nitpicking here but will maybe more clearly give the intent)

**Comment by vvo** [comment]
Tests are failing for now

**Comment by Haroenv** [comment]
build passes again, was failing because npm was down 🙃

**Pull request merged**